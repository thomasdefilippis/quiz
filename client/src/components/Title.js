import React from 'react';
import { AppBar, Typography } from '@material-ui/core';
import { makeStyles } from '@material-ui/core/styles';

const useStyles = makeStyles((theme) => ({
    root: {
      textAlign: 'center',
      backgroundColor: 'black'
    },

    appBar: {
    backgroundColor: '#cccccc'
    },

    title: {
        fontSize: '40px',
        color: '#009ACD'
    },
}));

const Title = () => {
    const classes = useStyles();
    return (
        <div className={classes.root}>
            <AppBar position="static" className={classes.appBar}>
                <Typography variant="h6" className={classes.title}>
                    Quiz
                </Typography>
            </AppBar>
        </div>
    )
}

export default Title;