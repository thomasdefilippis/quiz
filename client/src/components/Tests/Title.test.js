import Title from '../Title';
import { expectation } from 'sinon';

it("renders correctly", () => {
    const wrapper = shallow(    //shallow does not mount entire DOM but just gives a shallow version of it
        <Title />
    );

    expect(wrapper).toMatchSnapshot();
});

it("renders correctly again", () => {
    const wrapper = render(  
        <Title />
    );

    expect(wrapper).toMatchSnapshot();
});