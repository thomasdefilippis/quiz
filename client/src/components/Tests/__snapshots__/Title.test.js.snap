// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="makeStyles-root-1"
>
  <WithStyles(ForwardRef(AppBar))
    className="makeStyles-appBar-2"
    position="static"
  >
    <WithStyles(ForwardRef(Typography))
      className="makeStyles-title-3"
      variant="h6"
    >
      Quiz
    </WithStyles(ForwardRef(Typography))>
  </WithStyles(ForwardRef(AppBar))>
</div>
`;

exports[`renders correctly again 1`] = `
<div
  class="makeStyles-root-1"
>
  <header
    class="MuiPaper-root MuiAppBar-root MuiAppBar-positionStatic MuiAppBar-colorPrimary makeStyles-appBar-2 MuiPaper-elevation4"
  >
    <h6
      class="MuiTypography-root makeStyles-title-3 MuiTypography-h6"
    >
      Quiz
    </h6>
  </header>
</div>
`;
